### ISSUE TITLE
Provide documentation how to get information about if the test can get certified 

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/teamengine#323]
The information if a test can get certified should be automatically derived from the new report (EARL). E.g. a XPath is required how to get this information.

**Comment by dstenger** [comment]
Just the basic conformance classes are required to get certified.

**Comment by dstenger** [comment]
Skipped basic conformance classes are skipped during evaluation.

**Comment by keshav-nangare** [comment]
@dstenger

We already have an attribute in new HTML report which shows the test can be certified or not.

Results for session s0305
Test Name: wfs20
Test version: 1.30
Time: 2018-10-16T17:11:33.876Z
```

Test INPUT:
Tested Instance : http://cite.deegree.org/deegree-webservices-3.4.0/services/wfs200?service=WFS&request=GetCapabilities
xsd : http://cite.deegree.org/deegree-webservices-3.4.0/services/wfs200?service=WFS&version=2.0.0&request=DescribeFeatureType

Result: 
Passed Core Conformance Classes (Implementations passing these classes can be certified):No  <<====
Number of conformance classes tested: 13
Number of conformance class passed: 4
Number of conformance class failed: 2
```

@lgoltz 

Is this issue related to other implementation?

**Comment by lgoltz** [comment]
@keshav-nangare The XPath expression is required to get the information from the EARL report.

**Comment by dstenger** [comment]
@keshav-nangare 
We need following statement:
* One should return true or false (equals `Passed Core Conformance Classes (Implementations passing these classes can be certified)`).
* Description how to execute it (including file).

**Comment by keshav-nangare** [comment]
Proposed solution:

Add new element into earl report which will provide the status whether the test can be certified or not. This element contains the value true or false.
```
<cite:passedCoreConformanceClasses>false</cite:passedCoreConformanceClasses>
```

Xpath Expression to access this value: `//cite:TestRun/cite:passedCoreConformanceClasses`

**Comment by keshav-nangare** [comment]
Fixed with this commit: 99fc9c773635ddadab8f450e2fa9c630a46133b6

**Comment by lgoltz** [comment]
@keshav-nangare Please rename the element to `areCoreConformanceClassesPassed` (instead of `passedCoreConformanceClasses`)

**Comment by keshav-nangare** [comment]
Fixed with this PR #390.
